libvast/vast/view.hpp

```diff
@@ -234,7 +234,7 @@ class container_view_iterator
 
 public:
   container_view_iterator(container_view_ptr<T> x, size_t pos)
```

Comment:
Taking a `container_view_ptr` by value still manipulates the reference count. This increments the reference count only to store a raw pointer and then decrementing the counter again when leaving the ctor. I think taking a raw pointer as argument also signalizes best to the consumer that this iterator is storing a raw pointer, not a reference-counted one.